near-sdk/compilation_tests/schema_derive_invalids.stderr

```diff
@@ -84,7 +69,7 @@ error[E0277]: the trait bound `Inner: JsonSchema` is not satisfied
             i128
           and $N others
 note: required by a bound in `SchemaGenerator::subschema_for`
- --> $CARGO/schemars-0.8.15/src/gen.rs
+ --> $CARGO/schemars-0.8.13/src/gen.rs
```

Comment:
Remove your local Cargo.lock and re-run the test again. I expect this change to be resolved.